blocks/text.js

```diff
@@ -689,3 +689,90 @@ Blockly.Blocks['text_prompt'] = {
   mutationToDom: Blockly.Blocks['text_prompt_ext'].mutationToDom,
   domToMutation: Blockly.Blocks['text_prompt_ext'].domToMutation
 };
+
+Blockly.Blocks['text_count'] = {
+  /**
+   * Block for counting how many times one string appears within another string.
+   * @this Blockly.Block
+   */
+  init: function() {
+    this.jsonInit({
+      'message0': Blockly.Msg.TEXT_COUNT_MESSAGE0,
+      'args0': [
+        {
+          'type': 'input_value',
```

Comment:
The team is working on making these blocks more portable and more JSON-legal.  Things like Blockly.Msg.TEXT_COUNT_MESSAGE0 are currently being replaced with a reference notation (don't worry about that, we're just starting that process).  At the moment the process of porting blocks to Android and iOS is a manual copy/paste operation since human supervision is still required.